src/main/cpp/util/path_windows.cc

```diff
@@ -454,7 +454,11 @@ bool AsAbsoluteWindowsPath(const std::basic_string<char_type>& path,
     return false;
   }
   if (!IsRootOrAbsolute(*result, /* must_be_root */ false)) {
-    *result = GetCwdW() + L"\\" + *result;
+    if (result->empty() || (result->size() == 1 && (*result)[0] == '.')) {
+      *result = GetCwdW();
+    } else {
+      *result = GetCwdW() + L"\\" + *result;
+    }
   }
   if (!HasUncPrefix(result->c_str())) {
     *result = std::wstring(L"\\\\?\\") + *result;
```

Comment:
Just curious: Do relative paths still work when prefixed with `\\?\` ?

Reply:
Sorry - ignore that. I somehow thought that this method can also produce relative path (GitHub showed the diff in a strange way, until I refreshed the page and suddenly it's 10x as large)